_data/San97/San87.md

```diff
@@ -0,0 +1,5 @@
+name: Sneh Nitin Desai
+institution:Sardar Vallabhbhai Patel Institute of Technology, Vasad
+profile_pic: sneh.jpg
+quote: Success comes to those who not only understands but also acts on creating a better tomorrow. Change as it is the only constant.
```

Comment:
There's a 100 character limit on quotes, so you'll need to shorten this a bit (sorry!)